group_utils/javascript/automation/lib/javascript/community/groupUtils.js

```diff
@@ -0,0 +1,347 @@
+/**
+ * Utilities for working with openHAB groups in JavaScript.
+ * This code is not compatible with the GraalVM JavaScript add-on.
+ *
+ * Copyright (c) 2021 Florian Hotze under MIT License
+ * @florian-h05(https://github.com/florian-h05)
+ */
+
+(function (context) {
+  /**
+   * Imports.
+   */
+  var logger = Java.type('org.slf4j.LoggerFactory').getLogger('org.openhab.model.script.Rules.GroupUtils')
+  var Collectors = Java.type('java.util.stream.Collectors')
+
+  /**
+   * Private function to get the direct members of a group.
+   *
+   * @param {string} group Name of the openHAB group
+   * @param {string} characteristic Which characteristic of the members you get, valid: name (default), label, state
+   * @returns {Array} Given characteristic of direct members
+   */
+  function getMembers (group, characteristic) {
```

Comment:
By convention, a pseudo-private function like this (JavaScript doesn't support truly private functions) starts with `_`. `_getMembers`.